### PR TITLE
feat(args): implement arguments to command

### PR DESCRIPTION
**Pull request by stephenlautier**
Fixes https://github.com/sketch7/ngx.command/issues/7

### CODE REVIEW

File: src/command.directive.ts [hunk 1]
```diff
@@ -8,37 +8,77 @@ import {
...
+ *
  */
 @Directive({
 	selector: "[command]",
 })
 export class CommandDirective implements OnInit, OnDestroy {
-	@Input() command!: ICommand;
+	@Input("command") commandInput!: ICommand | CommandCreator | undefined;
 	@Input() commandOptions!: CommandOptions;
+	@Input() commandParams: any;
```
Comment (claylaut) [inline src/command.directive.ts]:
```suggestion
	@Input() commandParams: any[];
```

File: src/command-ref.directive.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+			const execFn = this.commandInput.execute.bind(hostComponent);
+			this.command = new Command(execFn, this.commandInput.canExecute, isAsync);
+		} else {
+			throw new Error("[ssvCommandRef] is not defined properly!");
+		}
+	}
+
+	ngOnDestroy() {
+		// console.log("[commandRef::destroy]");
```
Comment (claylaut) [inline src/command-ref.directive.ts]: I m not sure if you're leaving all logs commented, if not remove

File: .editorconfig [hunk 3]
```diff
@@ -9,4 +9,5 @@ indent_style = spaces
 indent_size = 2
```
Comment (claylaut) [inline .editorconfig]: You can add code owners file as well

File: src/command.directive.ts (hunk 1 above)
Comment (claylaut) [reply src/command.directive.ts]: I m not sure if this need to be an array:
since even if its one item it needs to be an array

Comment (stephenlautier) [reply src/command.directive.ts]: no, both of them are supported, i will improve doc, thanks. Will change it to `any | any[]`

File: .editorconfig (hunk 3 above)
Comment (stephenlautier) [reply .editorconfig]: added

**Pull request merged**